api/v1alpha1/reaper_types.go

```diff
@@ -76,6 +76,17 @@ type CassandraBackend struct {
 	AuthProvider AuthProvider `json:"authProvider,omitempty" yaml:"authProvider,omitempty"`
 }
 
+func (c *CassandraBackend) DatacenterName() string {
```

Comment:
When I originally wrote a lot of this, there was no integration or even planned integration with cass-operator. Now though we are tightly integrating with cass-operator which simplifies some things. In the `CassandraBackend` struct we can add a `CassandraDatacenter` property along these lines:

```go
type CassandraDatacenterRef struct {
        Name string `json:"name"`

        // If empty we could default the Reaper namespace.
        Namespace string `json:"namespace,omitempty"`
}

type CassandraBackend struct {
       CassandraDatacenter CassandraDatacenterRef `json:"cassandraDatacenter"`

       // otrher fields....
}
```

The `ClusterName` and `CassandraService` fields in `CassandraBackend` could then go away. This will eliminate the need for the `DatacenterName()` function.

When you update type definitions, be sure to run `make manifests` to regenerate updated CRDs.